packages/icons/icons.json

```
@@ -303,63 +303,63 @@
     {
         "displayName": "Inbox",
         "iconName": "inbox",
-        "tags": "folder, mail, file, message",
+        "tags": "folder, mail, file, message, drawer, container",
         "group": "file",
         "codepoint": 61951
     },
     {
         "displayName": "More",
         "iconName": "more",
-        "tags": "dots, three, extra, new, options",
+        "tags": "dots, three, extra, new, options, ellipses, break, pause, elipse, dot dot dot, suspense, see more, loading",
```

```suggestion
        "tags": "dots, three, extra, new, options, ellipses, break, pause, suspense, see more, loading",
```

- not sure about spelling of "elipse" / its relation to the mathematical shape
- "dot dot dot" is basically redundant with "dot"